### PR TITLE
is-714/chore: add copy and layout changes

### PR DESCRIPTION
**Pull request by alexanderleegs**
Fixes copy + layout changes on preview new editor modal and also changes scrolling to internal instead of external.

Also resolves is-785

Before:
Screenshot 2023-11-21 at 1 36 11 PM

After:
Screenshot 2023-11-21 at 1 37 09 PM

### CODE REVIEW

File: src/layouts/EditPage/MarkdownEditPage.tsx [hunk 1]
```diff
@@ -187,26 +187,33 @@ const PreviewModal = ({
...
+          <Text mb="1.5rem">
+            We’re introducing a new editor on IsomerCMS. Using this editor, you
+            can edit pages without using any Markdown or HTML. Explore what your
+            content looks like on the new editor.
+          </Text>
+          <Text>
+            You can toggle to use the new editor anytime on Page Settings.
+          </Text>
+          <Infobox my="1.5rem" variant="warning">
+            The current editor will be phased out by Q2 2024.
```
Comment (dcshzj) [inline src/layouts/EditPage/MarkdownEditPage.tsx:207]: nit: the "Q2 2024" should be in bold.

**Review by dcshzj — APPROVED** [review]
Non-blocking comments.

File: src/layouts/EditPage/MarkdownEditPage.tsx [hunk 2]
```diff
@@ -187,26 +187,33 @@ const PreviewModal = ({
...
   return (
-    <Modal isOpen={isOpen} onClose={onClose} size="xl">
+    <Modal isOpen={isOpen} onClose={onClose} size="4xl" scrollBehavior="inside">
       <ModalOverlay />
       <ModalContent p="1rem">
         <ModalHeader>
-          <Text textStyle="h3">New editing preview</Text>
+          <Text textStyle="h3">Preview Isomer’s new editor</Text>
         </ModalHeader>
         <ModalCloseButton />
```
Comment (dcshzj) [inline src/layouts/EditPage/MarkdownEditPage.tsx:196]: nit: the modal close button is too close to the edge, might need to add the padding similar to how it was done inside CreateMediaFolderModal

File: src/layouts/EditPage/MarkdownEditPage.tsx (hunk 1 above)
Comment (alexanderleegs) [reply src/layouts/EditPage/MarkdownEditPage.tsx]: Fixed in 759bd550!

File: src/layouts/EditPage/MarkdownEditPage.tsx (hunk 2 above)
Comment (alexanderleegs) [reply src/layouts/EditPage/MarkdownEditPage.tsx]: Fixed in 759bd550!

**Review by dcshzj — APPROVED** [review]

**Pull request merged**